### PR TITLE
2023 license update.

### PR DESCRIPTION
**Pull request by jackheslop96**

### CODE REVIEW

File: app/controllers/actions/CheckCancellationStatus.scala [hunk 1]
```diff
@@ -68,7 +68,7 @@ class CancellationStatusAction(
             html => Left(BadRequest(html))
           )
 
-      case Some(responseDeparture: ResponseDeparture) if validStatus.contains(responseDeparture.status) =>
```
Comment (ajchilts) [inline app/controllers/actions/CheckCancellationStatus.scala:71]: Did you intend to remove this?

File: app/forms/mappings/Formatters.scala [hunk 2]
```diff
@@ -89,7 +86,7 @@ trait Formatters {
       private val baseFormatter = stringFormatter(requiredKey)
 
       override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], A] =
-        baseFormatter.bind(key, data).right.flatMap {
```
Comment (ajchilts) [inline app/forms/mappings/Formatters.scala:92]: Guessing removing the `right` is a result of a library bump? As we have logic changes we will need to run regression before a deploy.

File: app/forms/mappings/LocalDateFormatter.scala [hunk 3]
```diff
@@ -52,10 +51,10 @@ private[mappings] class LocalDateFormatter(
     )
 
     for {
-      day   <- int.bind(s"$key.day", data).right
-      month <- int.bind(s"$key.month", data).right
-      year  <- int.bind(s"$key.year", data).right
-      date  <- toDate(key, day, month, year).right
+      day   <- int.bind(s"$key.day", data)
```
Comment (ajchilts) [inline app/forms/mappings/LocalDateFormatter.scala:54]: As above.

**Review by ajchilts — COMMENTED** [review]
Few comments @jackheslop96

File: app/forms/mappings/Formatters.scala (hunk 2 above)
Comment (jackheslop96) [reply app/forms/mappings/Formatters.scala:92]: Either is right-biased in Scala 2.13 so we get a bunch of warnings saying a .right can be removed

File: app/controllers/actions/CheckCancellationStatus.scala (hunk 1 above)
Comment (jackheslop96) [reply app/controllers/actions/CheckCancellationStatus.scala:71]: This doesn't actually change the behaviour as the first case is just the inverse of this. It was causing a 'not exhaustive pattern match' warning.

**Review by ajchilts — APPROVED** [review]
👍

**Pull request merged**